content/guides/virtualizing/digitalocean.md

```diff
@@ -0,0 +1,30 @@
++++
+type = "article"
+title = "Emulating Haiku on DigitalOcean"
+date = "2019-12-25T23:41:00.000Z"
+tags = []
++++
+
+DigitalOcean is one of the most popular KVM Cloud providers that allow the end user to upload raw disk images. As they don't allow booting from ISO, we will need to use VirtualBox or other virtualization software for installing Haiku. This guide assumes that you already have installed Haiku and you have a disk image (`*.vdi` for VirtualBox, `*.vmdk` for VMware, etc). If you don't have Haiku set up that way just yet, check out the [Virtualizing Haiku in VirtualBox](https://www.haiku-os.org/guides/virtualizing/virtualbox) guide. Remember that the hard disk size shouldn't exceed the disk size of your droplet!
```

Comment:
Your guide also assumes that the reader has experience using DigitalOcean. Please add another assumption note in this paragraph that this guide is also intended for those who are experienced with DigitalOcean.